envi/archs/arm/disasm.py

```diff
@@ -1482,26 +1509,25 @@ def p_uncond(opval, va):
             #clrex
             mnem = "clrex"
             olist =()
-            opcode = INS_CLREX
+            opcode = IENC_UNCOND_CLREX
```

Comment:
:) i don't think this is helpful.

IENC_ means "Instruction Encoding Scheme" (which made sense for the initial ARM breakdown, but less so for the Thumb instructions).
INS_ means "Instruction".